fluentgen/src/main/java/com/azure/autorest/fluent/mapper/FluentMapper.java

```diff
@@ -67,9 +87,96 @@ public FluentClient map(CodeModel codeModel, Client client) {
+                    String operationId = exampleStep.getOperationId();
+                    String[] oprs = operationId.split("_");
+                    String operationGroup = oprs[0];
+                    String operation = oprs[1];
+                    ProxyMethodExample example = exampleStep.getExample();
+                    FluentResourceCollection resourceCollection = findResourceCollection(fluentClient, operationGroup);
+
+                    // find collectionMethod
+                    Optional<FluentCollectionMethod> collectionMethodOptional = findCollectionMethod(resourceCollection, operation);
+                    if (collectionMethodOptional.isPresent()) {
+                        FluentCollectionMethod collectionMethod = collectionMethodOptional.get();
+                        FluentCollectionMethodExample collectionMethodExample = ExampleParser.parseMethodExample(resourceCollection, collectionMethod, example);
+                        FluentExampleTemplate.ExampleMethod exampleMethod = fluentExampleTemplate.generateExampleMethod(collectionMethodExample);
+                        setExampleStepFeatures(result, testCase, fluentStep, collectionMethodExample, exampleMethod);
+                    } else {
+                        // find resourceCreate
+                        Optional<ResourceCreate> createMethod = findResourceCreate(resourceCollection, operation);
+                        if (createMethod.isPresent()) {
+                            ResourceCreate create = createMethod.get();
+                            FluentResourceCreateExample createExample = ExampleParser.parseResourceCreate(resourceCollection, create, example);
+                            FluentExampleTemplate.ExampleMethod exampleMethod = fluentExampleTemplate.generateExampleMethod(createExample);
+                            setExampleStepFeatures(result, testCase, fluentStep, createExample, exampleMethod);
+                        } else {
+                            // find resourceUpdate
+                            Optional<ResourceUpdate> updateMethod = resourceCollection.getResourceUpdates().stream().filter(rc -> FluentUtils.exampleIsUpdate(rc.getMethodName()) && rc.getMethodName().equalsIgnoreCase(operation)).findFirst();
+                            if (updateMethod.isPresent()) {
+                                ResourceUpdate update = updateMethod.get();
+                                FluentResourceUpdateExample updateExample = ExampleParser.parseResourceUpdate(resourceCollection, update, example);
+                                if (updateExample == null) {
+                                    continue parseLiveTests;
+                                }
+                                FluentExampleTemplate.ExampleMethod exampleMethod = fluentExampleTemplate.generateExampleMethod(updateExample);
+                                setExampleStepFeatures(result, testCase, fluentStep, updateExample, exampleMethod);
+                            }
```

Comment:
Log a warning or throw exception if still not find?

Reply:
Maybe log and ignore this test case?

Reply:
OK

Reply:
Currently ignore unrecognized steps to let live test generate.